source/common/buffer/buffer_impl.h

```diff
@@ -17,6 +17,8 @@
 namespace Envoy {
 namespace Buffer {
 
+class OwnedImpl;
```

Comment:
Can remove the forward declaration since we no longer need to friend the class.

Reply:
right.